.openshiftio/application.yaml

```diff
@@ -65,6 +72,14 @@ objects:
   kind: BuildConfig
   metadata:
     name: rest-http
+    labels:
+      app.kubernetes.io/part-of: http-api-example
+      app.kubernetes.io/runtime-version: "SPRING_BOOT_VERSION"
+      app.kubernetes.io/runtime: snowdrop
```

Comment:
Is there a ticket opened for Openshift Devconsole to request them to display the Snowdrop icon ? @gytis

Reply:
I don't know if they show anything. We never had labels before

Reply:
I'd say we add the labels and then sort out the logos once the time comes.

Reply:
Lets add labels and if we see that logo doesn't get displayed , we open a ticket.